porcupine/plugins/filemanager.py

```diff
@@ -221,6 +242,8 @@ def is_NOT_project_root(path: Path) -> bool:
     # Doing something to an entire project is more difficult than you would think.
     # For example, if the project is renamed, venv locations don't update.
     # TODO: update venv locations when the venv is renamed
+    Command("Copy", None, is_NOT_project_root, copy),
+    Command("Paste", None, (lambda p: p.is_dir() and copy_path != ''), paste),
```

Comment:
Several suggestions:
- Use `<<Copy>>` and `<<Paste>>` virtual events which are Command+C on Mac, and Ctrl+C on windows and linux. Copy/pasting without Ctrl+C Ctrl+V just wouldn't feel right :)
- Use `None` as the "missing" value for `copy_path`
- Rename "Paste" to "Paste here", because the destination must be a directory

```suggestion
    Command("Copy", "<<Copy>>", is_NOT_project_root, copy),
    Command("Paste here", "<<Paste>>", (lambda p: p.is_dir() and copy_path is not None), paste),
```